templates/components/products/price-range.html

```diff
@@ -33,8 +56,47 @@
         {{/if}}
     </div>
 {{/and}}
+{{#and price.retail_price_range.min.without_tax price.retail_price_range.max.without_tax}}
+        <div class="price-section price-section--withoutTax rrp-price--withoutTax" {{#if schema_org}}itemprop="offers" itemscope itemtype="http://schema.org/Offer"{{/if}}>
+            {{lang 'products.retail_price'}}
+            <span data-product-rrp-price-without-tax class="price price--rrp">{{price.retail_price_range.min.without_tax.formatted}} - {{price.retail_price_range.max.without_tax.formatted}}</span>
+            {{#if schema_org}}
```

Comment:
I am curious about this meta content......I included it because it was already in the code that I was sampling but does it make sense for us to have these meta pricing properties for a retail price range?

I noticed that regarding an individual variant price, we do not include meta properties for a retail price - only the actual price.  So maybe these metas are safe to leave off of retail pricing or they should be added to the retail price for non-range pricing.